### PR TITLE
[FIX] Update travis configuration to fix CI

### PR DESCRIPTION
**Pull request by yajo**
Update it to latest template.

@Tecnativa TT19224

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,83 +1,25 @@
...
 env:
   global:
-  - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  # Set this variable to some version existing as linux-generic build on
-  # https://github.com/wkhtmltopdf/wkhtmltopdf/releases
-  # if you need to install wkhtmltopdf
-  # - WKHTMLTOPDF_VERSION="0.12.4"
-  # Set the above to install a `wkhtmltopdf` version that is not the one provided
-  # by the `pov-wkhtmltopdf` repo.
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
```
Comment (pedrobaeza) [inline .travis.yml]:
```suggestion
  - VERSION="11.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
```

File: .travis.yml [hunk 2]
```diff
@@ -1,83 +1,25 @@
...
+  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
   - TESTS="1" ODOO_REPO="OCA/OCB"
-# either use the two lines above or the two below. Don't change the default if
-# it's not necessary (it is only necessary if modules in your repository can't
-# be installed in the same database. And you get a huge speed penalty in your
-# tests)
-#  - TESTS="1.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1"
-#  - TESTS="1.0" ODOO_REPO="OCA/OCB" UNIT_TEST="1"
-
-virtualenv:
```
Comment (pedrobaeza) [inline .travis.yml:79]: Isn't this needed?

**Comment by rafaelbn** [comment]
@Yajo travis should be green? it is red :red_circle:

**Comment by rafaelbn** [comment]
@Yajo maybe is the same as https://github.com/OCA/project-agile/pull/20#issuecomment-533001197

File: .travis.yml (hunk 2 above)
Comment (yajo) [reply .travis.yml:79]: It's not in the template anymore, so I guess not.

**Review by rafaelbn — APPROVED** [review]
:+1:

**Comment by rafaelbn** [comment]
We can merge this! :+1: this repo is empty in this moment and travis is happy :green_apple:

**Pull request merged**